src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -102,6 +102,15 @@ public bool IsAnalysisSupported(IEnumerable<AnalysisLanguage> languages)
         }
 
         public void ExecuteAnalysis(string path,
+            IEnumerable<AnalysisLanguage> detectedLanguages,
+            IIssueConsumer consumer,
+            IAnalyzerOptions analyzerOptions,
+            IAnalysisStatusNotifier statusNotifier,
+            CancellationToken cancellationToken) =>
+            TriggerAnalysisAsync(path, detectedLanguages, consumer, analyzerOptions, statusNotifier, cancellationToken)
```

Comment:
The product changes are quite simple:
* previously, the request creation was done before the analysis was triggered on a separate thread.
* now, Execute immediately triggers a separate thread to do all of the work, including creating the request.

Unfortunately, changing the tests wasn't as straightforward.